addons/ofxGui/src/ofxBaseGui.cpp

```diff
@@ -225,22 +227,28 @@ void ofxBaseGui::setPosition(float x, float y){
 	setNeedsRedraw();
 }
 
-void ofxBaseGui::setSize(float w, float h){
+void ofxBaseGui::setSize(float w, float h, bool callback){
 	b.width = w;
```

Comment:
let's remove this callbacks parameters, it's a pattern that we use sometimes in OF and it's really confusing to read.

this:

```
gui.sizeChangedE.disable()
gui.setSize(40,40)
gui.sizeChangedE.enable()
```

is more verbose but it's clear what it means, but this:

```
gui.setSize(40,40,false);
```
it's really cryptic and forces you to go to the original headers to see the meaning of that parameter, another possibility would be something like:

```
gui.setSizeWithoutEvent(40,40);
```

but it would need to be public and since it's something we only need internally to avoid recursive calls i think the first syntax is ok